Bottom: a2bad0041a3a687a737b91130d4aa5f79488363e
Top:    a540c52a792312c36c9cacfbfb23e1cf5c477e0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 23:44:10 +0100

Refresh of hid-gtk-do-not-run-a-main-loop

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index a25976a..31caa2e 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -622,8 +622,6 @@ node_get_node_from_name (gchar * node_name, LibraryMenuType ** node_net)
    */
   ghid_netlist_window_create (gport);
 
-  /* XXX */
-
   /* Now walk through node entries of each net in the net model looking for
      |  the node_name.
    */
@@ -882,8 +880,6 @@ ghid_get_net_from_node_name (gchar * node_name, gboolean enabled_only)
    */
   ghid_netlist_window_create (gport);
 
-  /* XXX */
-
   /* If no netlist is loaded the window doesn't appear. */
   if (netlist_window == NULL)
     return NULL;
